src/services/ui/src/api/submissionService.ts

```diff
@@ -142,16 +142,13 @@ export const buildAttachmentKeyValueArr = (
 export const urlsToRecipes = (
   urls: PreSignedURL[],
   attachments: AttachmentKeyValue[],
-  authority: Authority,
 ): UploadRecipe[] =>
   urls.map((obj, idx) => ({
     ...obj, // Spreading the presigned url
     data: attachments[idx].file, // The attachment file object
     // Add your attachments object key and file label value to the attachmentTitleMap
     // for this transform to work. Else the title will just be the object key.
-    title:
-      attachmentTitleMap(authority)?.[attachments[idx].attachmentKey] ||
-      attachments[idx].attachmentKey,
+    title: attachments[idx].attachmentKey,
```

Comment:
so much better that we're live-mapping so mapped values aren't set in stone at submission 👍🏼 this cleans it up nicely